static/app/views/performance/newTraceDetails/traceModels/traceTree.tsx

```diff
@@ -1659,6 +1672,44 @@ export class TraceTree extends TraceTreeEventDispatcher {
     throw new Error('Unknown trace type');
   }
 
+  get title(): TraceTree.Title {
```

Comment:
This replicates the logic used by the explorer endpoint to assign a name to the trace. Here's a [link](https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_traces.py#L857-L895) to it.